code/modules/food_and_drinks/drinks/drinks/cans.dm

```diff
@@ -66,7 +66,7 @@
 				handle_bursting(user)
 	else
 		to_chat(H, "<span class='warning'>You need to hold [src] in order to shake it.</span>")
-	return ..()
+	return
```

Comment:
If you try to drag the can while it's on he floor, you just get this message, which makes cans un-dragable by humans while on the floor. I think this should probably be
```dm
else
	return ..()
```
I know it's not ideal but, since we're co-opting the drag key, we kinda need this.